wiki_indexer/connector.py

```diff
@@ -127,13 +126,14 @@ def index(self, id=None):
         if id:
             with self.engine.connect() as conn:
                 select_statement = self.table.select().where(self.table.c.id == id)
-                set = conn.execute(select_statement).fetchone()
+                set = conn.execute(select_statement).fetchall()
```

Comment:
set is reserved keyword. 

Also why you replaced `fetchone` by `fetchall` ?

Reply:
So fetchone returns one object that is not subscriptable, while functions that works with this object requires subscriptable object.